src/components/ReviewCard/ReviewCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./ReviewCard.css";
+import * as BiIcons from "react-icons/bi";
+import * as FaIcons from "react-icons/fa";
+import { updateVotes } from "../../utils/api";
+import { organiseDate } from "../../utils/organiseDate";
+
+const ReviewCard = ({ review }) => {
+  const [date, time] = organiseDate(review.created_at);
+
+  return (
+    <li className="review-card">
+      <h3>{review.title}</h3>
+      <img
+        className="game-image"
+        src={review.review_img_url}
+        alt={review.title}
```

Comment:
Personally I'd say the titles aren't always a good indication for the alt text
Your images are a little large too, I'd get the styling sorted for those, max-height etc...